benchmarking/benchmark.py

```diff
@@ -51,207 +23,44 @@
 N_CORES = [2**i for i in range(MAX_CORES.bit_length())]
 if MAX_CORES > N_CORES[-1]:
     N_CORES.append(MAX_CORES)
-COUNT_TYPES = [False, True]
-SPARSE_TYPES = [False, True]
-PLOT_DIR = "./benchmark_times_plotted"
-SCORE_DIR = "./benchmark_times_saved"
-
-
-def get_times_skfp(
-    X: pd.DataFrame, transformer_function: FingerprintTransformer, **kwargs
-):
-    if "count" in kwargs:
-        count = kwargs["count"]
-        print(f" - - count: {count}")
+PLOT_DIR = "benchmark_times/benchmark_times_plotted"
+SCORE_DIR = "benchmark_times/benchmark_times_saved"
 
-    n_jobs = kwargs["n_jobs"]
-    print(f" - - n_jobs: {n_jobs}")
 
+def get_times_skfp(X: pd.DataFrame, transformer_constructor: Callable, **kwargs):
```

Comment:
Add return type `np.ndarray`. Mypy has to be able to run this directory.

Reply:
And this is not `transformer_constructor`, this is `transformer_cls` of type `Type` (as all classes)